app/code/Magento/Captcha/Test/Mftf/Data/CaptchaConfigData.xml

```diff
@@ -33,6 +33,12 @@
         <data key="label">Contact Us</data>
         <data key="value">contact_us</data>
     </entity>
+    <entity name="StorefrontCaptchaOnCheckoutConfigData">
```

Comment:
@DmitryTsymbal Please add a new entity changes at the end of the file

Reply:
@shikhamis11 , thanks for noticing. I have moved new entity to the end of the file. Please check